### PR TITLE
Fix minification computation and CSS its

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/minify/AverageLineLengthCalculator.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public boolean isLineInHeaderComment(String line) {
   }
 
   private boolean isFirstLineInHeaderComment(String line) {
-    if (line.startsWith("/*")) {
+    if (line.startsWith("/*") && (!line.contains("*/") || line.endsWith("*/"))) {
```
Comment (vilchik-elena) [inline sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/minify/AverageLineLengthCalculator.java:79]: tbh I am not sure about the impact of this fix on those computations for JS. Are we sure there will be no change in behaviour?

**Review by vilchik-elena — APPROVED** [review]
LGTM 
But please update PR title and commit to contain the fix for minification.
Also I am not 100% that the fix you did for minification is harmless.

File: sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/minify/AverageLineLengthCalculator.java (hunk 1 above)
Comment (saberduck) [reply sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/minify/AverageLineLengthCalculator.java:79]: tests are OK, do we trust our tests? :)

Comment (vilchik-elena) [reply sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/minify/AverageLineLengthCalculator.java:79]: Who is guarding the guards?)

**Pull request merged**